.github/workflows/helm_pr_deployment.yml

```diff
@@ -0,0 +1,98 @@
+name: Deploy PR to Kubernetes
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
```

Comment:
According to the GH documentation, those 3 values are consistent with the set of default values. I propose to omit them for clarity reasons here